app/routes/wow.weekly-price-group-delta-recommended.tsx

```
@@ -105,8 +118,46 @@ export default function RecommendedWeeklyPriceGroupDelta() {
+                  className={`
+                    ${
+                      activeTab === 'patch-cycles'
+                        ? 'bg-blue-600 text-white shadow-lg'
+                        : 'bg-gray-200 text-gray-700 hover:bg-gray-300 dark:bg-gray-700 dark:text-gray-300 dark:hover:bg-gray-600'
+                    }
+                    whitespace-nowrap py-3 px-6 rounded-lg font-semibold text-lg transition-all duration-150
+                  `}
+                  aria-current={
+                    activeTab === 'patch-cycles' ? 'page' : undefined
+                  }>
+                  Patch Cycles
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)